CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
```

Comment:
"If you prefer the v6 behavior"?

Reply:
v5 here is correct.
This means after upgrading to styled components v6, vendor prefix is no longer enabled by default like in v5, if you prefer the v5(old) behavior, follow v6 migration guide to fallback to v5 behavior.